lecture_4/02_bets_api/internal/api/controllers/controller.go

```diff
@@ -0,0 +1,92 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+// Controller implements handlers for web server requests.
+type Controller struct {
+	betRepository BetRepository
+}
+
+const wonStatus = "won"
+const lostStatus = "lost"
+const activeStatus = "active"
```

Comment:
FYI, this can be written also as 
```suggestion
const (
    wonStatus = "won"
    lostStatus = "lost"
    activeStatus = "active"
)
```
Also, usually const declarations are usually placed immediately after imports.

Reply:
I totally forgot that that is an option 😄 
Thank you!